changelogs/fragments/e2e-manage-vm-tests.yml

```diff
@@ -0,0 +1,5 @@
+---
+major_changes:
+  - manage_vm - adding manage vm e2e tests using molecule testing framwork. this includes
+                tests for update, delete and adding a vm to the vcenter and validates the outcome of the changes.
+title: provision_vm_test
```

Comment:
The file name should start with the PR number 23-e2e-manage-vm-tests.yml

Regarding the title I'm not sure if necessary

Reply:
tnx! checking